### PR TITLE
feat: add (loadType/deleteValue/pvc)  in Cluster New , and add deleteValue in Tenant New

### PR DESCRIPTION
**Pull request by TianWuwt**
…e in Tenant New

## Summary

feat(): add loadType 、deleteValue、pvc  in Cluster New ,add deleteValue in Tenant New
image
image
image

### CODE REVIEW

File: ui/src/pages/Cluster/New/Observer.tsx [hunk 1]
```diff
@@ -173,7 +192,21 @@ export default function Observer({ storageClasses, form }: ObserverProps) {
           </Col>
         </Row>
         <p className={styles.titleText}>storage</p>
-
+        <p className={styles.subTitleText}>存储卷配置</p>
```
Comment (powerfooI) [inline ui/src/pages/Cluster/New/Observer.tsx:195]: Why not use `CustomFormItem`? I don't think this form item for pvc protection is special among other items.

File: ui/src/pages/Cluster/New/Observer.tsx [hunk 2]
```diff
@@ -59,7 +73,12 @@ export const TooltipItemContent = ({ item }) => {
   );
 };
 
-export default function Observer({ storageClasses, form }: ObserverProps) {
+export default function Observer({
+  storageClasses,
+  form,
+  pvcValue,
```
Comment (powerfooI) [inline ui/src/pages/Cluster/New/Observer.tsx:79]: Why not put this item into the `form`?

File: ui/src/pages/Tenant/New/BasicInfo.tsx [hunk 3]
```diff
@@ -182,6 +183,36 @@ export default function BasicInfo({
...
+                <Option key={item.value} value={item.value}>
+                  {item.label}
+                </Option>
+              ))}
+            </Select>
+          </Form.Item>
+        </Col>
+
+        <Col span={8}>
+          <Form.Item name={['deleteVal']} label={<></>}>
```
Comment (powerfooI) [inline ui/src/pages/Tenant/New/BasicInfo.tsx:209]: Why leave an empty label here?

File: ui/src/pages/Cluster/New/Observer.tsx (hunk 1 above)
Comment (TianWuwt) [reply ui/src/pages/Cluster/New/Observer.tsx:195]: Style does not match

File: ui/src/pages/Tenant/New/BasicInfo.tsx (hunk 3 above)
Comment (TianWuwt) [reply ui/src/pages/Tenant/New/BasicInfo.tsx:209]: Style positioning to avoid dislocation

File: ui/src/pages/Cluster/New/Observer.tsx (hunk 1 above)
Comment (TianWuwt) [reply ui/src/pages/Cluster/New/Observer.tsx:195]: Style does not match

File: ui/src/constants/index.ts [hunk 4]
```diff
@@ -473,3 +473,26 @@ export const EVENTSTABLE_STATUS_LIST = [
...
+    label: 'HTAP',
+  },
+  {
+    value: 'OLAP',
+    label: 'OLAP',
+  },
+  {
+    value: 'KV',
+    label: 'KV',
+  },
```
[the remark this answers is not in the document]
Comment (TianWuwt) [reply ui/src/constants/index.ts:497]: done

File: ui/src/pages/Cluster/New/Observer.tsx (hunk 2 above)
Comment (TianWuwt) [reply ui/src/pages/Cluster/New/Observer.tsx:79]: get the data here

Comment (powerfooI) [reply ui/src/pages/Cluster/New/Observer.tsx:79]: Why not get the value from the `form`?

**Review by powerfooI — APPROVED** [review]

**Pull request merged**